pages/login_page.php

```diff
@@ -0,0 +1,19 @@
+<?php 
+
+require_once '../src/include/database.php';
+require_once '../src/include/nav_functions.php';
+require_once '../src/include/include.php';
+
+$categories = get_categories();
+
+$login_data = [];
+
+$login_page = include_template('../src/templates/login.php', $login_data );
```

Comment:
Если данных, нет, то их можно вообще не передавать и выкинуть. Т.е.
```suggestion
$login_page = include_template('../src/templates/login.php');
```

Reply:
Исправлено